File: results/py/OVERLAPPING_SUM_TWO_ARRAY.cpp
int f_filled ( int A [ ] , int B [ ] , int n ) {
  int Hash [ n ] ;
  for ( int i = 0 ;
  i != n ;
  i ++ ) {
    Hash [ A [ i ] ] ++ ;
    Hash [ B [ i ] ] ++ ;
  }
  int Sum = 0 ;
  for ( int x : Hash ) {
    if ( Hash [ x ] == 1 ) Sum += x ;
  }
  return Sum ;
}

File: TransCoder_tokenized_test_set_functions/java/OVERLAPPING_SUM_TWO_ARRAY.java
static int findSum ( int [ ] A, int [ ] B, int n ) {
  HashMap < Integer, Integer > hash = new HashMap < > ( );
  for ( int i = 0;
  i < n;
  i ++ ) {
    if ( hash . containsKey ( A [ i ] ) )
        hash . put ( A [ i ], 1 + hash . get ( A [ i ] ) );
    else hash . put ( A [ i ], 1 );
    if ( hash . containsKey ( B [ i ] ) )
        hash . put ( B [ i ], 1 + hash . get ( B [ i ] ) );
    else
        hash . put ( B [ i ], 1 );
  }
  int sum = 0;
  for ( Map . Entry entry : hash . entrySet ( ) ) {
    if ( Integer . parseInt ( ( entry . getValue ( ) ) . toString ( ) ) == 1 )
        sum += Integer . parseInt ( ( entry . getKey ( ) ) . toString ( ) );
  }
  return sum;
}